source/tutorials/nix-shell-in-shebang.md

```diff
@@ -0,0 +1,58 @@
+# Reproducible interpreted scripts
+
+In this tutorial, you will learn how to use the [nix shell] to create reproducible interpreted scripts.
+We will write a script requesting latest releases of NixOS/nixpkgs GitHub repository, convert it from XML to JSON and pretty-print the result.
```

Comment:
We should present the script as given, explain what it does, and then work along that.